File: Front-end/health-mind-system-frontend/src/shared/types/dtos/PsicologoEdicao.dto.ts
import { StatusCargoEnum } from "../../domain/enums/status-cargo.enum";
import { StatusRoleEnum } from "../../domain/enums/status-role.enum";

export interface PsicologoEdicaoDTO {
    id?: string;
    nome: string;
    email?: string;
    cpfCnpj: string;
    statusCargo: StatusCargoEnum;
    statusRole: StatusRoleEnum;
    crp: string;
    especialidade: string;
    valorConsulta: number;
    regenerarCredenciais?: boolean;
}

File: Back-end/HealthMindBackend/HealthMindBackend.API/DTOs/PsicologoEdicaoDTO.cs
using HealthMindBackend.Domain.Enums;

namespace HealthMindBackend.API.DTOs
{
    public class PsicologoEdicaoDTO
    {
        public String Id { get; set; }
        public String Nome { get; set; }
        public String? Email { get; set; }
        public String? Senha { get; set; }
        public String CpfCnpj { get; set; }
        public StatusCargoEnum StatusCargo { get; set; }
        public StatusRoleEnum StatusRole { get; set; }
        public String Crp { get; set; }
        public String Especialidade { get; set; }
        public Decimal ValorConsulta { get; set; }
        public Boolean RegenerarCredenciais { get; set; }
    }
}
